clients/filter/filter.go

```diff
@@ -149,8 +175,15 @@ func (c *Client) GetDimensionOptions(filterID, name string) (opts []DimensionOpt
 }
 
 // CreateJob creates a filter job and returns the associated filterJobID
-func (c *Client) CreateJob(instanceID string) (string, error) {
-	fj := Model{InstanceID: instanceID, State: "created"}
+func (c *Client) CreateJob(instanceID string, names []string) (string, error) {
```

Comment:
`CreateJob` should probably be `CreateBlueprint` to align with API spec. It would be confusing for developer bouncing back and forth from web services and api to find the naming inconsistent. Remember a filter job doesn't exist anymore.